### PR TITLE
Exceptions, Unit Test

### PR DESCRIPTION
**Pull request by wzalazar**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

## Description of Changes

Fix https://github.com/poetapp/node/issues/361

This PR introduces

- unit test for Exceptions

### CODE REVIEW

File: src/Exceptions.test.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import 'Extensions/Error'
+import { describe } from 'riteway'
+
+describe('Exceptions NoMoreEntriesException', async (should: any) => {
+  const { assert } = should('')
+  const noMoreEntriesException = new NoMoreEntriesException('noMoreEntriesException')
+  const parseError = JSON.parse(JSON.stringify(noMoreEntriesException))
+
+  {
+    assert({
```
Comment (lautarodragan) [inline src/Exceptions.test.ts]: This test is only testing the `new` keyword.

**Comment by wzalazar** [comment]
@lautarodragan ready to review ;)

**Comment by lautarodragan** [comment]
@wzalazar isn't this testing `Extensions/Error` rather than `NoMoreEntriesException`?

**Comment by wzalazar** [comment]
@lautarodragan well `NoMoreEntriesException` extends of `Error`, is similar, but we need to do the test for coverage purposes and ensure the behavior is correct

**Comment by wzalazar** [comment]
@lautarodragan I didn't understand your point. For my side, I need to test this file because we have 

screen shot 2018-12-07 at 11 47 41

instead of 100%

screen shot 2018-12-07 at 11 48 06

**Comment by wzalazar** [comment]
Commenting the file `Extensions/Error.test.ts`, I get the same coverage when I have the 2 tests. So I can delete the test `Extensions/Error.test.ts` if you want

screen shot 2018-12-07 at 18 35 33

When I comment the two test obviously any has coverage

screen shot 2018-12-07 at 18 36 11

**Comment by wzalazar** [comment]
ready @lautarodragan, I removed the assertion

**Pull request merged**